src/versions/util.js

```diff
@@ -1,5 +1,6 @@
 // from most recent versions to oldest ones
 const apiVersions = [
+  '2020-06-12', // TODO: adapt the version number
```

Comment:
Given the other breaking changes pending, you can probably set this to `2020-07-20` in one month, for now.